intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -2401,6 +2401,33 @@ def convert_date(value):
     }
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/sinkhole-dns-report/
+accessible_sinkhole_dns = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip', validate_ip),
+        ('source.port', 'port', convert_int)
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'host'),
+        ('source.asn', 'asn', convert_int),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('extra.', 'type'),
```

Comment:
I would use this field as `extra.dns_query_type`. (also in use in the anubisnetworks parser)